doc/developers/maintainer.rst

```diff
@@ -81,10 +141,31 @@ Making a release
 
        $ python setup.py sdist
 
+   - You can also test a binary dist build using::
+
+       $ python setup.py bdist_wheel
+
+   - You can test if PyPi is going to accept the package using::
+
+       $ twine check dist/*
+
+   You can run ``twine check`` after step 5 (fetching artifacts) as well.
```

Comment:
Let's move it there instead then.

Reply:
I put it here cause step 5 is after creating the tag. Ideally once the tag is created, there shouldn't be more necessary changes.